### PR TITLE
Use mean instead of sum for mask loss

### PR DESCRIPTION
**Pull request by PhysSong**
This is one more thing I mentioned before.

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -83,6 +84,8 @@ def loss_for_permutation(
         # If mask_order is not provided, use the best one
         if input_permutation is None:
             loss += best_loss
+    # Sum -> Average
+    loss /= B
```
Comment (yycho0108) [inline train.py:88]: Does this just work as-is? We don't have to change reduction parameters in any of the above loss functions?

**Review by yycho0108 — APPROVED** [review]
LGTM, though I'm not 100% sure if this actually computes what we want.

File: train.py (hunk 1 above)
Comment (PhysSong) [reply train.py:88]: We calculate the loss for each single batch in `loss_for_permutation` function and then collect them, so it works.

Comment (yycho0108) [reply train.py:88]: ok.
